extension/src/popup/components/manageAssets/ManageAssetRows/index.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { useState } from "react";
+import StellarSdk, { Account } from "stellar-sdk";
+import { useDispatch, useSelector } from "react-redux";
+import { Loader } from "@stellar/design-system";
+
+import { AppDispatch } from "popup/App";
+
+import { navigateTo } from "popup/helpers/navigate";
+
+import { ROUTES } from "popup/constants/routes";
+
+import { publicKeySelector } from "popup/ducks/accountServices";
+import { settingsNetworkDetailsSelector } from "popup/ducks/settings";
+import {
+  ActionStatus,
+  getAccountBalances,
+  signFreighterTransaction,
+  submitFreighterTransaction,
+  transactionSubmissionSelector,
+} from "popup/ducks/transactionSubmission";
+
+import { AssetIcon } from "popup/components/account/AccountAssets";
+
+import { CURRENCY } from "@shared/api/types";
+
+import "./styles.scss";
+
+export type ManageAssetCurrency = CURRENCY & { domain: string };
+
+interface ManageAssetRowsProps {
+  assetRows: ManageAssetCurrency[];
+  setErrorAsset: (errorAsset: string) => void;
+}
+
+const getCodeIssuerStr = (assetCode: string, assetIssuer: string) =>
```

Comment:
❓ can we call this `getCanonicalFromAsset` or something similar? there's the opposite func in the [helpers](https://github.com/stellar/freighter/blob/6cae2173abd40ab35d82038ffc30644e69f240ad/extension/src/helpers/stellar.ts#L41) now, could we move this one next to it in that file? (I'm sure we'll re-use this at some point)

I've been using the term `canonical` when referring to `<asset code>:<issuer>`, I've noticed we're not 100% consistent referring to it that way across projects but I think it's a good way to refer to it, and is used in [SEPs](https://github.com/stellar/stellar-protocol/pull/313) (thought not all idt)

Reply:
Yup, good call